gui/src/app/StanSampler/StanSampler.ts

```diff
@@ -12,6 +12,17 @@ export type SamplingOpts = {
     seed: number | undefined
 }
 
+export const isSamplingOpts = (x: any): x is SamplingOpts => {
```

Comment:
Do you mind also adding a use of this check to SPAnalysisQueryLoading? There is a TODO there for it

Reply:
Done.